assets/runtime/env-defaults

```diff
@@ -24,6 +24,9 @@ fi
 GITLAB_SSH_HOST=${GITLAB_SSH_HOST:-$GITLAB_HOST}
 GITLAB_SSH_PORT=${GITLAB_SSH_PORT:-$GITLAB_SHELL_SSH_PORT} # for backwards compatibility
 GITLAB_SSH_PORT=${GITLAB_SSH_PORT:-22}
+SSH_HOST_KEYS_DIR=${SSH_HOST_KEYS_DIR:-${GITLAB_DATA_DIR}/ssh/}
+SSH_HOST_KEYS_DIR=${SSH_HOST_KEYS_DIR%/}/
```

Comment:
Duplication of line 27. I think you should write line 27 like `SSH_HOST_KEYS_DIR=${SSH_HOST_KEYS_DIR:-${GITLAB_DATA_DIR}/ssh}` and the removal of the trailing backslash in line 28 is obsolete in my opinion.  Because you remove it and add the slash again this is a little bit confusing 😅 .